packages/techdocs-common/src/stages/generate/techdocs.ts

```diff
@@ -66,24 +66,36 @@ export class TechdocsGenerator implements GeneratorBase {
   public async run({
     directory,
     dockerClient,
+    expectedResultDir,
```

Comment:
hmm why "expected"? 🤔   not just `outputDir`?

Reply:
Oh! I couldn't use `resultDir` since it's being used in the function body. Then I added a prefix to it, as it came to find.